pkg/kube/cluster-init.sh

```diff
@@ -721,6 +690,18 @@ check_cluster_config_change() {
               # redo the debugger user role binding since certs are changed
               rm /var/lib/debuguser-initialized
             fi
+
+            if [ "$is_bootstrap" = "true" ]; then
+              # provision this node to cluster mode with bootastrap config
+              logmsg "privision config file for bootstrap node"
+              provision_cluster_config_file true
+            else
+              # provision this node to cluster mode with server config, will wait for join-ip to be ready
+              # we transition to cluster mode, and we need to wait for the bootstrap server status
+              logmsg "privision config file for non-bootstrap node"
+              provision_cluster_config_file true
```

Comment:
@naiming-zededa Why is provision_cluster_config_file in the condition if its called with the same arguments either way?

Reply:
good catch, removed the condition. It was taken different params before, but that was not correct.